.pfnci/wheel-windows/_flexci.ps1

```diff
@@ -49,7 +49,7 @@ function ActivateCUDA($version) {
     } elseif ($version -eq "11.7") {
         $Env:CUDA_PATH = $Env:CUDA_PATH_V11_7
     } elseif ($version -eq "11.x") {
-        $Env:CUDA_PATH = $Env:CUDA_PATH_V11_7
+        $Env:CUDA_PATH = $Env:CUDA_PATH_V11_8
```

Comment:
Please add this as well.

```
    } elseif ($version -eq "11.8") {
        $Env:CUDA_PATH = $Env:CUDA_PATH_V11_8
```

Also note: This file should be in sync with the one in `cupy/cupy`.